uportal-war/src/main/webapp/media/skins/respondr/common/less/content.less

```diff
@@ -293,30 +293,30 @@ section.emergency-alert {
 }
 
 /* large screens show all 6 columns. */
-@media only screen and (min-width: 769px) {
+@media only screen and (min-width: @screen-sm-min) {
     .portal-page-column.col-md-2 {
         width: 16.6666667%;
         float: left;
     }
 }
 
-@media only screen and (max-width: 769px) {
+@media only screen and (max-width: @screen-sm-max) {
     /* collapse from 6 to 3 columns */
     .portal-page-column.col-md-2 {
         width: 33.3%;
         float: left;
     }
 }
 
-@media only screen and (max-width: 600px) {
+@media only screen and (max-width: @screen-sm-max - 167) {
```

Comment:
having a breakpoint `167px` off from a bootstrap breakpoint feels off.
Should this be something like the average between size small and size extra small?

``` less
@media only screen and (max-width: (@screen-sm-max + @screen-xs-max) / 2) {}
```
With the default bootstrap values this would evaluate to `624px`.